scripts/data_files/driver_jsons/mbedtls_test_opaque_driver.json

```diff
@@ -0,0 +1,20 @@
+{
+    "prefix":       "mbedtls_test",
+    "type":         "opaque",
+    "location":     "0x7fffff",
+    "dependency":   "defined(PSA_CRYPTO_DRIVER_TEST)",
+    "headers":      ["test/drivers/test_driver.h"],
+    "capabilities": [
+        {
+            "_comment":     "The mbedTLS opaque driver supports import key/export key/export_public key",
+            "depend_on":    "defined(PSA_CRYPTO_DRIVER_TEST)",
+            "entry_points": ["import_key", "export_key", "export_public_key"]
+        },
+        {
+            "_comment":     "The mbedTLS opaque driver supports copy key/ get builtin key",
+            "depend_on":    "defined(PSA_CRYPTO_DRIVER_TEST)",
```

Comment:
I'll be changing the names to two different ones (h_depends_on and depends_on), or it can be further nuanced as h_depends_on and c_depends_on (header depends on and capability depends on). This should not be a pure driver dependency and could vary based on each capability, for example an opaque driver as such could require a a base set of dependencies, while individual capabilities could be more granularly defined, for which the driver developer might not want multiple different versions of drivers, but configurable capabilities.